arm-compute/2016-03-30/swagger/compute.json

```diff
@@ -4307,6 +4307,11 @@
           "type": "boolean",
           "description": "Specifies whether the latest model has been applied to the virtual machine."
         },
+        "vmId": {
```

Comment:
Although the property is "readOnly", unless customers update the generated client SDKs to the next version the client will ignore this property as it is not aware about it.  Since the generated client does not know about it, it will throw away the property during deserialization and the property will not appear during serialization as well. Hence the GET| PUT pipeline scenario will not be affected. (I assume this holds true for all the AutoRest generated clients).

IMO, this does not seem to be a breaking change. However, I would like to know what other people think about it?
@fearthecowboy @lmazuel @veronicagg @johanste - what do you guys think?